packages/react-components/src/templates/ResearchOutputPage.tsx

```diff
@@ -21,16 +25,21 @@ const ResearchOutputPage: React.FC<ResearchOutputPageProps> = ({
   documentType,
   researchOutputData,
   isEditMode,
+  publishingEntity = 'Team',
   ...formProps
```

Comment:
I think this file just makes things more complex and will need to be duplicated due to the new Working Group Report type. I think we should just move all the logic from Research Output Form into this file and rename it to ResearchOutputTeamForm.tsx . We can then have another called ResearchOutputWorkingGroupForm.tsx which will have the special handling around that